src/Uno.UI/UI/Xaml/WindowManagerInterop.wasm.cs

```diff
@@ -579,13 +587,21 @@ internal static string GetAttribute(IntPtr htmlId, string name)
 		#region ClearAttribute
 		internal static void RemoveAttribute(IntPtr htmlId, string name)
 		{
+
+#if NET7_0_OR_GREATER
```

Comment:
@rafael-rosa-knowcode Was this added on purpose? @ebariche does this ring a bell?

Reply:
No and it won't work !

<img width="593" alt="image" src="https://user-images.githubusercontent.com/33458222/236138739-7e273846-2b5b-4282-b5a2-050e6c60c713.png">

Reply:
@rafael-rosa-knowcode : Undo all changes to this file.

1 - Neither change is needed.
2 - SetAttribute needs a specialized binding to avoid array allocations.
3 - RemoveAttribute is not implemented correctly.